nastaveni/nastaveni.php

```diff
@@ -152,6 +150,8 @@
 // odpočítané tvrdé údaje podle dat
 define('REG_GC', mezi(REG_GC_OD, REG_GC_DO));
 define('REG_AKTIVIT', mezi(REG_AKTIVIT_OD, REG_AKTIVIT_DO));
+define('LIMIT', mezi(REG_AKTIVIT_OD, HROMADNE_ODHLASOVANI));
+define('LIMIT2', mezi(HROMADNE_ODHLASOVANI, HROMADNE_DRUHE_ODHLASOVANI));
```

Comment:
Slovo limit působí velmi abstraktně – z toho mi plyne, že neumíme ten interval nějak jednoznačněji pojmenovat, což ukazuje na to, že to pro nás není ani žádná uchopitelnější entita => spíš bychom to tedy neměli dávat do konstant, ale radši počítat až ve finance.php (roli hraje i to, že se to používá jen na jednom místě).

(mluvím o obou konstantách LIMIT)

Reply:
no spíš mám obecně problém vymýšlet názvy... co PRVNI_ODHLASENI a DRUHE_ODHLASENI?

Reply:
Já bych to asi nechal v nastavení nicméně, pokud na tom trváš. Není problém to přesunout.